app/views/teachers/reviews/_form.html.erb

```diff
@@ -12,23 +11,51 @@
   <% end %>
 
   <div>
-    <%= form.label :student_id, style: "display: block" %>
-    <%= form.select :student_id, @user.group_members.all.map(&:user).pluck(:name, :id)
- %>
   </div>
-
-  <div>
-    <%= form.label :review, style: "display: block" %>
-    <%= form.text_area :review %>
+  <div class="field is-horizontal">
+    <div class="field-label is-normal">
+      <%= form.label :student_id, class: "label" %>
+    </div>
+    <div class="field-body">
+      <div class="field">
+        <div class="select">
+          <%= form.select :student_id, @my_students&.pluck(:name, :id), required: true, class: "input" %>
+        </div>
+      </div>
+    </div>
   </div>
 
-  <div>
-    <%= form.label :learning_resource_id, style: "display: block" %>
-    <%= form.select :learning_resource_id, @user.learning_resources&.pluck(:name, :id).prepend(["", ""]), selected: review.learning_resource_id || ["", ""] %>
+  <div class="field is-horizontal">
+    <div class="field-label is-normal">
+      <%= form.label :review, class: "label" %>
+    </div>
+    <div class="field-body">
+      <div class="field">
+        <p class="control">
+          <%= form.text_area :review, required: true, class:"input" %>
+        </p>
+      </div>
+    </div>
   </div>
 
+  <div class="field is-horizontal">
+    <div class="field-label is-normal">
+      <%= form.label :learning_resource_id, class: "label" %>
+    </div>
+    <div class="field-body">
+      <div class="field">
+        <div class="select">
+          <%= form.select :learning_resource_id, @user.learning_resources&.pluck(:name, :id).prepend(["None", ""]), selected: review.learning_resource_id || ["None", ""] , class: "input"%>
```

Comment:
I'd change the `prepend` for the native [include_blank](https://api.rubyonrails.org/v5.1.7/classes/ActionView/Helpers/FormOptionsHelper.html)